app/feature/feature-profile/src/main/kotlin/com/hedvig/android/feature/profile/tab/ProfileGraph.kt

```diff
@@ -44,8 +45,8 @@ fun NavGraphBuilder.profileGraph(
       deepLinks = listOf(
         navDeepLink { uriPattern = hedvigDeepLinkContainer.profile },
       ),
-      enterTransition = { MotionDefaults.fadeThroughEnter },
-      exitTransition = { MotionDefaults.fadeThroughExit },
+      enterTransition = { fadeIn() }, // todo: sure here?
+      exitTransition = { fadeOut() },
```

Comment:
The ones in MotionDefaults do use a tween spec
```
animationSpec = tween(
      durationMillis = FadeThroughDuration.ForIncoming,
      delayMillis = FadeThroughDuration.ForOutgoing,
      easing = MotionTokens.EasingStandardCubicBezier,
    ),
```
while the default for this is spring() iirc.

It is subtle, but the reason this is important there is that when we are fading from one top-level destination to another, we want to not have the two contents ever overlap with each other.
That means that as the animation is going, first the first screen fades out completely before the new one starts fading in.
With this, there is a slight overlap where both, screenshot for reference:
![image](https://github.com/user-attachments/assets/a1173e74-ac27-4a4c-9486-18baf076ec7e)

So yeah we want to copy this over to the new DS too to have access to it.